gradle-plugin/src/main/kotlin/org/mozilla/android/MozillaPlugin.kt

```diff
@@ -6,9 +6,34 @@ package org.mozilla.android
 
 import org.gradle.api.Plugin
 import org.gradle.api.Project
+import org.gradle.api.artifacts.repositories.MavenArtifactRepository
+import java.net.URI
 
 class MozillaPlugin : Plugin<Project> {
 
     override fun apply(project: Project) {
+        with(project) {
```

Comment:
Can we encapsulate this in a function? Ideally, `apply` should read like a high-level feature list:
```kotlin
override fun apply(project: Project) {
    addAppServicesMavenRepository()
    // ...
}
```